include/exadg/acoustic_conservation_equations/spatial_discretization/spatial_operator.cpp

```diff
@@ -326,6 +327,29 @@ SpatialOperator<dim, Number>::apply_inverse_mass_operator(BlockVectorType &
   inverse_mass_velocity.apply(dst.block(block_index_velocity), src.block(block_index_velocity));
 }
 
+template<int dim, typename Number>
+double
+SpatialOperator<dim, Number>::calculate_time_step_cfl() const
+{
+  // In case of mixed-orders use the maximum fe_degree and the corresponding
+  // quadrature rule.
+
+  // The time-step size is not adapted every time-step. Thus, we are using
+  // a constant function to pass in the speed of sound, even though it is
+  // be possible to optimize calculate_time_step_cfl_local() for this case.
```

Comment:
"it is be possible"

Reply:
[3eeb568](https://github.com/exadg/exadg/pull/620/commits/3eeb568630fefc005322180097ce3ee411e0ace0)